tests/typ/compiler/shorthand.typ

```diff
@@ -1,7 +1,7 @@
 // Test shorthands for unicode codepoints.
 
 ---
-The non-breaking space~does work, soft-?hypen also does.
+The non-breaking space~does work, soft-?hyphen also does.
```

Comment:
Don't forget to update the reference image of this test 🙂

Reply:
I wasn't sure how I'm supposed to do this. Would I just copy the image over from the generated outputs?

Reply:
The image also should be optimized. You can also use the test CLI for this. See the tests Readme for details.